src/lib/openapi/spec/create-update-context-schema.ts

```diff
@@ -0,0 +1,38 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { legalValueSchema } from './legal-value-schema';
+
+export const createUpdateContextSchema = {
```

Comment:
Maybe this could be `upsertContextSchema`.

Reply:
Realized that `contextFieldSchema` is already compliant with what we need in those request bodies, so I refactored to only use `contextFieldSchema` and keep things simple.